packages/functional-tests/lib/polyfills.js

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This didn't work, probably going to see about refactoring `lib/otp.ts` in settings a bit instead.